### ISSUE TITLE
Navigate to Screens on Other Tabs

### ISSUE THREAD

**Issue by zachrnolan** [parkerdan/SampleNavigation#3]
First of all, thank you for this example repo, great stuff!

Do you have to explicitly include every screen inside each tabs StackNavigator?

Say I have the following tabs with screens:

- Tab 1
  - Tab 1 Screen 1
  - Tab 1 Screen 2
- Tab 2
  - Tab 2 Screen 1
  - Tab 2 Screen 2
- Tab 3
  - Tab 3 Screen 1
  - Tab 3 Screen 2

I see how I can navigate from Tab 1 Screen 1 to Tab 1 Screen 2, but what if I want to navigate to Tab 3 Screen 2 from within Tab 1?

Basically I'd like to freely be able to navigate to any screen regardless of the tab it belongs to and remain within the tab of origination.

Thanks!

**Comment by parkerdan** [comment]
If you want to remain in Tab One, you have to include every possible Tab One Screen in the Navigation Configurations.  You can re-use screen components, but the routeConfigs need need to be defined.

If you are taking over with Redux, you can always dispatch a custom action and toggled tabs and navigate deep within another tab from anywhere...   

https://medium.com/@parkerdan/react-navigation-with-complete-redux-state-management-tab-bar-and-multiple-navigators-ed30a69d9a4d#.jdersaop4
